Sources/Model/TransferMethod/HyperwalletTransferMethodQueryParam.swift

```diff
@@ -28,34 +28,32 @@ public class HyperwalletTransferMethodQueryParam: QueryParam {
     }
 
     /// Representation of the transfer method status
-    ///
-    /// - activated: Filter by activated bank accounts
-    /// - deActivated: Filter by deActivated transfer methods
-    /// - invalid: Filter only invalid bank accounts
-    /// - lostOrStolen: Filter only lostOrStolen prepaid cards
-    /// - preActivated: Filter by preActivated prepaid cards
-    /// - suspended: Filter only suspended prepaid cards
-    /// - verified: Filter only verified bank accounts
     public enum QueryStatus: String {
+        /// Filter by activated bank accounts
```

Comment:
change bank accounts to transfer methods

Reply:
Changed